insights/parsers/ps.py

```diff
@@ -342,3 +342,42 @@ def parse_content(self, content):
     def children(self, ppid):
         """list: Returns a list of dict for all rows with `ppid` as parent PID"""
         return [row for row in self.data if row['PPID'] == ppid]
+
+
+add_filter(Specs.ps_aexww, "COMMAND")
+
+
+@parser(Specs.ps_aexww)
+class PsAexww(Ps):
+    """
+    Class to parse the output of ``ps aexww`` command
+
+    Sample input data::
+
+          PID TTY      STAT   TIME COMMAND
+            1 ?        Ss     0:08 /usr/lib/systemd/systemd --switched-root --system --deserialize 22
+            2 ?        S      0:00 [kthreadd]
+            3 ?        S      0:00 [ksoftirqd/0]
+            5 ?        S<     0:00 [kworker/0:0H]
+            7 ?        S      0:01 [migration/0]
+        24477 ?        Ssl    6:56 /usr/bin/openshift-router PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin HOSTNAME=node1 RELOAD_INTERVAL=10s
+
+    Examples:
+        >>> type(ps_aexww)
+        <class 'insights.parsers.ps.PsAexww'>
+        >>> ps_aexww.data[5]['ENVIRON']['RELOAD_INTERVAL']
+        '10s'
+    """
+    command_name = 'COMMAND'
+    user_name = 'PID'
```

Comment:
`user_name` is not `PID`.
As per the definition of `user_name`:
~~~
``user_name`` is the name of the first column from the header of ps output,
the subclass must override it correspondingly
~~~

Would you think it's reasonable to rename it as `first_column`?

Reply:
And since there is no `user` info provided in this `ps_aexwww`, you should reimplement the `users` method, to raise an Exception or others.